Stream-Two/Unit03-Testing/challenges/challenge2/vending_machine.py

```diff
@@ -0,0 +1,16 @@
+from decimal import *
+
+coins = [1, .50, .20, .10, .05, .02, .01]
+
+
+def give_change(amount):
+    change = []
+    # The amount is converted to a string then a decimal
+    amount = Decimal(str(amount))
+    for coin in coins:
+        coin = Decimal(str(coin))
+        while coin <= amount:
+            # No floating point errors as we are using only decimals
+            amount -= coin
+            change.append(float(coin))
```

Comment:
why are we using floats here? We should be able to work with Decimals throughout